src/App.tsx

```diff
@@ -1,25 +1,93 @@
-import React, { useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
+import { Todo } from './react-app-env';
+import { getAllTodos } from './apis/api';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 const App: React.FC = () => {
   const [
     selectedUserId,
-    // setSelectedUserId,
+    setSelectedUserId,
   ] = useState(0);
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const loadTodos = useCallback(
+    async () => {
+      const loadedTodos = await getAllTodos();
+
+      setTodos(loadedTodos);
+
+      return loadedTodos;
+    },
+    [],
+  );
+
+  const loadSelectedUser = useCallback(
+    (newSelectedUserId: number) => {
+      setSelectedUserId(newSelectedUserId);
+    },
+    [selectedUserId],
+  );
```

Comment:
if there is no user info your apps selection part collapses. You should put a try catch here. You can try the user #17 to see.